Meme/Meme/MemeParser.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  MemeParser.swift
+//  Meme
+//
+//  Created by Mesfin Bekele Mekonnen on 7/29/19.
+//  Copyright © 2019 Mesfin Bekele Mekonnen. All rights reserved.
+//
+
+import Foundation
+
+struct MemeParser {
```

Comment:
This should probably implement a protocol like `Parser`
```

Reply:
Yep, was thinking the same. I will do this soon.